master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/IMetadataHandler.java

```diff
@@ -82,4 +82,6 @@ void handleWorkerEvent(
       int workerEventTypeValue, List<WorkerInfo> workerInfoList, String requestId);
 
   void handleUpdatePartitionSize();
+
+  void handleApplicationMeta(String appId, String secret);
```

Comment:
Could we introduce `ApplicationMeta` to consturct metadata of application including appId, secret, and user identifer in future?

Reply:
Sure. I made this interface generic. It accepts `ApplicationMeta` now.